tests/python/pants_test/reporting/test_reporting_integration.py

```diff
@@ -154,3 +163,46 @@ def test_epilog_to_stderr(self, quiet_flag):
     self.assert_success(pants_run)
     self.assertIn('Cumulative Timings', pants_run.stderr_data)
     self.assertNotIn('Cumulative Timings', pants_run.stdout_data)
+
+  def test_zipkin_reporter(self):
+    with http_server(ZipkinHandler) as port:
+      endpoint = "http://localhost:{}".format(port)
+      command = [
+        '--reporting-zipkin-endpoint={}'.format(endpoint),
+        '--reporting-zipkin-encoding=json',
+        'cloc',
+        'src/python/pants:version'
+      ]
+
+      pants_run = self.run_pants(command)
+      self.assert_success(pants_run)
+
+      num_of_traces = len(ZipkinHandler._traces)
+      self.assertEqual(num_of_traces, 1)
+
+      trace = ZipkinHandler._traces[-1]
+      main_span = self.find_span_by_name(trace, 'main')
```

Comment:
Call this `main_span_ids`, because it's a list of IDs, not a single span? And maybe call `find_span_by_name` instead `find_span_ids_by_name`?